{"name": "test_register_with_wrong_credentials", "status": "passed", "description": "../features/register.feature: User Registration with wrong credentials", "start": 1596403689661, "stop": 1596403691818, "uuid": "3845efb7-dab5-49a4-991c-a20869b407ab", "historyId": "4d229c7a669ed31647536670586dab6e", "testCaseId": "d2c1000855a4d98598169968aee5d07f", "fullName": "tests.step_defs.test_register#test_register_with_wrong_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures"}, {"name": "tag", "value": "wrong-credentials"}, {"name": "tag", "value": "bddapi"}, {"name": "parentSuite", "value": "tests.step_defs"}, {"name": "suite", "value": "test_register"}, {"name": "host", "value": "DESKTOP-QE471D0"}, {"name": "thread", "value": "8040-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.step_defs.test_register"}]}